app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+def create_sorted_variables(*args) -> dict:
+    sorted_variables = {"mutable": [], "immutable": []}
+
+    for arg in args:
+        if isinstance(arg, (list, set, dict)):
+            sorted_variables["mutable"].append(arg)
+        else:
+            sorted_variables["immutable"].append(arg)
```

Comment:
Instead of using an if-else statement, you could use a ternary operator to make your code more concise. Here is an example: 'sorted_variables['mutable' if isinstance(arg, (list, set, dict)) else 'immutable'].append(arg)'.